app/src/main/java/net/squanchy/settings/SettingsFragment.kt

```diff
@@ -95,6 +95,15 @@ class SettingsFragment : PreferenceFragment() {
         }
 
         setupWifiConfigPreference()
+
+        val disableAnalyticsPreferences = findPreference(getString(R.string.disable_analytics_key))
+        disableAnalyticsPreferences.setOnPreferenceChangeListener { _, _ ->
```

Comment:
Not to do in this PR, but we likely want a dialog saying we only collect data through Crashlytics and Analytics because we want to fix bugs and we're not sharing/reselling data, and ask confirmation before disabling

Reply:
Let's open a PR for that?

Reply:
Yup, although it should be more generically "finalise analytics opt-out UX" I think

Reply:
#612

Reply:
Now that I think of it, should we also add a feature to sync this setting across user's devices if they're signed in?

Reply:
Or better, maybe we should have an epic to synchronise user settings across devices, and this would be one of them.

Reply:
I think we can rely on auto backup for that, which IIRC is enabled. We should test it through, but to be entirely honest, I don't think it's that important.